tests/wpt/mozilla/tests/mozilla/non-invertible-transform.html

```diff
@@ -0,0 +1,26 @@
+<!doctype html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <title>Shouldn't panic when transform is non-invertible</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <style>
+        #test {
+            width: 100px;
+            height: 100px;
+            transform: scale(0);
+        }
+    </style>
+</head>
+<body>
+<div id="test"></div>
+<script>
+    test(function() {
```

Comment:
This needs to be an async test, or it will complete before the setTimeout executes.